packages/runtime/src/useCases/transport/relationshipTemplates/RelationshipTemplateMapper.ts

```diff
@@ -17,6 +17,8 @@ export class RelationshipTemplateMapper {
             createdByDevice: template.cache.createdByDevice.toString(),
             createdAt: template.cache.createdAt.toString(),
             forIdentity: template.cache.forIdentity?.toString(),
+            password: template.password,
+            pin: template.pin,
```

Comment:
this should be combined to only password

- "password" for passwords
- "pin:1234" for pins

Reply:
Do you mean "pw:password" (or something like that)? Also would integrators like this if they want to do something with the password in their own systems?

Reply:
nah, password can be the default.

Reply:
argh

Reply:
yeah, then password: